README.md

```diff
@@ -36,22 +36,35 @@ IL weaver for adding nullability annotations to .NET Framework and .NET Standard
 
 ```xml
 <PropertyGroup>
-  <!-- Specifies the version of this rewriter to use -->
-  <TunnelVisionLabsReferenceAssemblyAnnotatorVersion>1.0.0-alpha.50</TunnelVisionLabsReferenceAssemblyAnnotatorVersion>
-  <!-- Specifies the version of Microsoft.NETCore.App.Ref to obtain nullability information from -->
-  <AnnotatedReferenceAssemblyVersion>3.0.0-preview9-19423-09</AnnotatedReferenceAssemblyVersion>
+  <!-- Specifies the version of this rewriter to use. -->
+
+  <TunnelVisionLabsReferenceAssemblyAnnotatorVersion>1.0.0-alpha.69</TunnelVisionLabsReferenceAssemblyAnnotatorVersion>
```

Comment:
I think the blank line was meant to go after each property?

Reply:
Absolutely, thanks! I have no idea what happened.